tests/local.rs

```diff
@@ -209,6 +209,25 @@ async fn stale_element(mut c: Client, port: u16) -> Result<(), error::CmdError>
     }
 }
 
+async fn select_by_index(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let url = sample_page_url(port);
+    c.goto(&url).await?;
+    let mut select_element = c.find(Locator::Css("#select1")).await?;
+
+    // Get first display text
+    let initial_text = select_element.html(false).await?;
+
+    // Select second option
+    select_element.clone().select_by_index(1).await?;
+
+    // Get display text after selection
+    let text_after_selecting = select_element.html(false).await?;
+
+    // Make sure that it's changed
+    assert_ne!(initial_text, text_after_selecting);
```

Comment:
It'd probably be good to assert that the text here is in fact the correct text. Maybe use `value` on the `select` itself to check that it indeed has the right option selected. Might also be a good idea to check that the other select did _not_ change.